test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterDuplicateMovementReferenceNumberSpec.scala

```diff
@@ -0,0 +1,267 @@
+
+        def performAction(): Future[Result] = controller.enterDuplicateMrn()(FakeRequest())
+
+        val (session, _, _) = sessionWithClaimState(None)
+
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(session.copy(journeyStatus = None))
+        }
+
+        checkIsRedirect(
+          performAction(),
+          baseRoutes.StartController.start()
+        )
+
+      }
+    }
+
+    "Duplicate MRN page" must {
+      def performAction(): Future[Result] = controller.enterDuplicateMrn()(FakeRequest())
+
+      "show the title" in {
+        val mrn             = MRN("10ABCDEFGHIJKLMNO0")
+        val answers         = CompleteMovementReferenceNumberAnswer(Right(mrn))
+        val (session, _, _) = sessionWithClaimState(Some(answers))
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(session)
+        }
+
+        val doc = Jsoup.parse(contentAsString(performAction()))
+
+        doc.select("h1").text should include(messageFromMessageKey("enter-duplicate-movement-reference-number.title"))
+
+      }
+
+      "Fail if the same Entry Number is submitted" in {
+        val entryNumber     = EntryNumber("123456789A12345678")
+        val answers         = CompleteMovementReferenceNumberAnswer(Left(entryNumber))
+        val (session, _, _) = sessionWithClaimState(Some(answers))
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(session)
+        }
+        val result          = controller.enterDuplicateMrnSubmit()(
+          FakeRequest().withFormUrlEncodedBody("enter-movement-reference-number" -> "123456789A12345678")
+        )
+        status(result) shouldBe BAD_REQUEST
+      }
+
+      "Fail on an Entry Number journey and an MRN is submitted" in {
+        val entryNumber     = EntryNumber("123456789A12345678")
+        val answers         = CompleteMovementReferenceNumberAnswer(Left(entryNumber))
+        val (session, _, _) = sessionWithClaimState(Some(answers))
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(session)
+        }
+        val result          = controller.enterDuplicateMrnSubmit()(
+          FakeRequest().withFormUrlEncodedBody("enter-movement-reference-number" -> "10ABCDEFGHIJKLMNO0")
+        )
+        status(result) shouldBe BAD_REQUEST
+      }
+
+      "Fail if the same MRN is submitted" in {
+        val mrn             = MRN("10ABCDEFGHIJKLMNO0")
+        val answers         = CompleteMovementReferenceNumberAnswer(Right(mrn))
+        val (session, _, _) = sessionWithClaimState(Some(answers))
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(session)
+        }
+        val result          = controller.enterDuplicateMrnSubmit()(
+          FakeRequest().withFormUrlEncodedBody("enter-movement-reference-number" -> "10ABCDEFGHIJKLMNO0")
+        )
+        status(result) shouldBe BAD_REQUEST
```

Comment:
You should check the content of this result, and see if the error message appears.